probe_src/probe_py/cli.py

```diff
@@ -116,4 +116,86 @@ def dump(
                     print(op.data)
                 print()
 
+@app.command(
+    context_settings=dict(
+        ignore_unknown_options=True,
+    ),
+)
+def ssh(
+        ssh_args: list[str],
+        debug: bool = typer.Option(default=False, help="Run verbose & debug build of libprobe"),
+):
+    """
+    Wrap SSH and record provenance of the remote command.
+    """
+
+    if "-i" not in ssh_args:
+        typer.secho("RSA key not defined", fg=typer.colors.RED)
+        raise typer.Abort()
```

Comment:
This wrapper should work whether or not the user supplies an SSH key. The OpenSSH client has a way of searching for a default key if none is specified.